examples/stable/mysql/README.md

```diff
@@ -128,7 +121,7 @@ s3-profile-drnw9   2m
 
 # Create Actionset
 # Please make sure the value of profile and blueprint matches with the names of profile and blueprint that we have created already
-$ kanctl create actionset --action backup --namespace kanister --blueprint mysql-blueprint --deployment mysql-test/mysql-release --profile mysql-test/s3-profile-drnw9 --secrets mysql=mysql-test/mysql-release
+$ kanctl create actionset --action backup --namespace kasten-io --blueprint mysql-blueprint --statefulset mysql-test/mysql-release-master --profile mysql-test/s3-profile-drnw9 --secrets mysql=mysql-test/mysql-release
```

Comment:
this should be in kanister namespace right. I dont think we are creating kasten-io namespace in this readme.

Reply:
Right, at one of the places, `kasten-io` was being used for namespace. Will update these.